x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/validate.ts

```diff
@@ -22,19 +20,6 @@ import type { RuleParams } from '../../schemas/rule_schemas';
 import type { LegacyRulesActionsSavedObject } from '../../rule_actions/legacy_get_rule_actions_saved_object';
 import { internalRuleToAPIResponse } from '../../schemas/rule_converters';
 
-export const transformValidate = (
-  rule: PartialRule<RuleParams>,
-  ruleExecutionSummary: RuleExecutionSummary | null,
-  legacyRuleActions?: LegacyRulesActionsSavedObject | null
-): [RulesSchema | null, string | null] => {
-  const transformed = transform(rule, ruleExecutionSummary, legacyRuleActions);
-  if (transformed == null) {
-    return [null, 'Internal error transforming'];
-  } else {
-    return validateNonExact(transformed, rulesSchema);
-  }
-};
-
 export const newTransformValidate = (
```

Comment:
If we are getting rid of the original `transformValidate`, couldn't we rename `newTransformValidate` to `transformValidate`?

Reply:
Yeah, updated to `transformValidate`